### ISSUE TITLE
suggestion

### ISSUE THREAD

**Issue by vearlen** [swirldev/swirl#829]
Hi!
The beauty of swirl is self-pace learning but I think of drawbacks is it's one direction only. After you finished the lessons, as far as I understood there is no way to see an overview (all what you've learned) and that I think would be great. Right now in order to recall any specific command user have to re-do the whole lesson and reach that point consequently. 
I think it would be good to have an overview of commands been sent during lessons or the whole lesson with results as a code file for users to able to make notes and recall what happened. 

Cheers,
Ilya

**Comment by cpgitgig135** [comment]
You can take snapshots with a screen capture program. Then you will be able to view all of the situations you encountered and then remember what you learned.

**Comment by vearlen** [comment]
Yes, for sure. It's not a problem even to re-run the program, however I thought it would be useful, if possible to give an option to have .md file or something like this from each course.

**Comment by cpgitgig135** [comment]
Yes, I suppose that swirl can provide a summary of commands, an optional refresher section for people that want to refresh/recall what they have learned without having to go thru all of the steps again (as you mentioned).
